packages/plugin-ext/src/plugin/scm.ts

```diff
@@ -635,13 +642,14 @@ class SourceControlImpl implements theia.SourceControl {
     ) {
         this._inputBox = new ScmInputBoxImpl(plugin, this.proxy, this.handle);
         this.proxy.$registerSourceControl(this.handle, _id, _label, _rootUri);
+        this.plugin = plugin;
     }
 
     private createdResourceGroups = new Map<SsmResourceGroupImpl, Disposable>();
     private updatedResourceGroups = new Set<SsmResourceGroupImpl>();
 
     createResourceGroup(id: string, label: string): SsmResourceGroupImpl {
-        const group = new SsmResourceGroupImpl(this.proxy, this.commands, this.handle, id, label);
+        const group = new SsmResourceGroupImpl(this.proxy, this.commands, this.handle, this.plugin, id, label);
```

Comment:
Not in scope for this PR, but shouldn't it be `ScmResourceGroupImpl`?

Reply:
Yeah, looks like a typo.
I can add it to this PR.